static/src/stylesheets/module/_social.scss

```diff
@@ -212,3 +212,50 @@ category: Common
     visibility: visible;
     opacity: 1;
 }
+
+// Testing static social on mobile
+
+.social-fixed {
+    height: 40px;
+    position: fixed;
```

Comment:
@phamann I've tested this on browsers that don't support position: fixed using browserstack. So that's iOS 4 and Android 2.3 and below, and they don't display any problems, though do we even run JavaScript for them? 

There doesn't seem to be an easy way to check if position: fixed is supported. This is a rather huge chunk of code: http://kangax.github.io/cft/#IS_POSITION_FIXED_SUPPORTED